openboxes.dbml

```diff
@@ -1,2719 +1,2429 @@
-Project openboxes {
-  database_type: 'MySQL'
-  Note: 'Data model for latest development version of OpenBoxes'
-}
-
-Table "DATABASECHANGELOG" {
-  "ID" varchar(63) [not null]
-  "AUTHOR" varchar(63) [not null]
-  "FILENAME" varchar(200) [not null]
-  "DATEEXECUTED" datetime [not null]
-  "MD5SUM" varchar(32) [default: NULL]
-  "DESCRIPTION" varchar(255) [default: NULL]
-  "COMMENTS" varchar(255) [default: NULL]
-  "TAG" varchar(255) [default: NULL]
-  "LIQUIBASE" varchar(10) [default: NULL]
-
-Indexes {
-  (ID, AUTHOR, FILENAME) [pk]
-}
-}
-
-Table "DATABASECHANGELOGLOCK" {
-  "ID" int(11) [pk, not null]
-  "LOCKED" tinyint(1) [not null]
-  "LOCKGRANTED" datetime [default: NULL]
-  "LOCKEDBY" varchar(255) [default: NULL]
-}
-
 Table "address" {
-  "id" char(38) [pk, not null, default: ""]
-  "version" bigint(20) [not null]
+  "id" varchar(255) [pk, not null]
```

Comment:
Bummer. I just noticed that Hibernate's schema-export does do exactly what our database migrations do, so I'm going to need to run the migrations (maybe on a testcontainer) and then use mysqldump to get a more precise DDL file to convent to DBML.